test/pdftotext.test.js

```diff
@@ -24,7 +24,7 @@ describe('pdftotext', function(){
     it('should throw exception', function(){
       var ptext = new pdftotext(__dirname + '/pdfs/invalidfile.pdf');
       function fn() {
-        var stdout = ptext.getTextSync();
+        return ptext.getTextSync();
```

Comment:
Odd because I don't see any differences locally with or without converting it to string. Also locally the tests don't fail but on the build this specific test fails.